test/test_images/scheduler_receiver/main.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2019 Google LLC
```

Comment:
nit: 2020

Reply:
wondering if we can pass in an ENV variable saying which source it is, and then just have one image, with an if, else if, else if, and so on... Seems that we are duplicating a bunch of main.go files, and we can just have one...
No need to do in this PR though... just a thought

Reply:
Yep. We can try create another issue to investigate how to do that